examples/kotlin-multiplatform/build.gradle

```diff
@@ -13,12 +13,20 @@ allOpen {
     annotation("org.openjdk.jmh.annotations.State")
 }
 
+def distinguishAttribute = Attribute.of('kotlinx-benchmark-distinguishAttribute', String)
+
 kotlin {
     jvm {
         compilations.create('benchmark') { associateWith(compilations.main) }
     }
-    js('jsIr', IR) { nodejs() }
-    js('jsIrBuiltIn', IR) { nodejs() }
+    js('jsIr', IR) {
+        nodejs()
+        attributes.attribute(distinguishAttribute, 'jsIr')
+    }
+    js('jsIrBuiltIn', IR) {
+        nodejs()
+        attributes.attribute(distinguishAttribute, 'jsIrBuiltIn')
+    }
```

Comment:
I was thinking about fixing the "targets are not distinguishable" issue by having a single JS target and adding a compilation. e.g.
```
    js(IR) {
        nodejs()
        compilations.create('builtInExecutor') {
            associateWith(compilations.main)
        }
    }
```

Considering the fact that declaring multiple targets of the same kind [won't be supported soon](https://youtrack.jetbrains.com/issue/KT-59316), this seems like a better long-term solution.

WDYT?